cs-many-resources/Dummy.cs

```diff
@@ -0,0 +1,15 @@
+using Pulumi;
```

Comment:
I'm a bit unsure about this being in the `examples` repo - that repo is a user-facing set of examples, and this is not a real example. 

If we do decide we really need this to live in `examples`, I'd suggest nesting it inside some "infrastructure" or "misc" like folder so it's not as user-facing.

Reply:
@stack72 thoughts on where to find a home for this?

1. new `pulumi-benchmarks` repo
2. put into `pulumi/pulumi`

Moving the code is trivial, but I'm a little unsure about cloning the YAML Github Action definitions that run the code, I think you mentioned Paul that those are really managed from some central code-generating repository, was it https://github.com/pulumi/ci-mgmt ? Could you give me a few pointers so I get it done the way we like it.

Reply:
For the immediate need right now I can definitely nest this so as not to distract the users looking for actual examples.